Gemfile

```diff
@@ -11,7 +11,14 @@ else
   gem "rails", "~> #{rails_version}"
 end
 
-gem "rack-test"
+group :test do
+  if RUBY_VERSION == "1.9.3"
```

Comment:
Ok, dropped support for 1.9.3, dropped `pry-debugger` for `pry-byebug` since debugger doesn't support 1.9.3, and updated `travis.yml` to build against ruby 2.x as well as the new Rails 4.2. Should be good to go now.